lib/srv/uacc/uacc_linux.go

```diff
@@ -195,6 +182,24 @@ func Close(utmpPath, wtmpPath string, tty *os.File) error {
 	}
 }
 
+// getDefaultPaths sets the default paths for utmp and wtmp files if passed empty.
+func getDefaultPaths(utmpPath, wtmpPath string) (string, string) {
```

Comment:
Nit: Use named return args for better clarity.
```suggestion
func getDefaultPaths(utmpPath, wtmpPath string) (utmpPath, wtmpPath string) {
```

Reply:
This is not going to work:
```
# github.com/gravitational/teleport/lib/srv/uacc
lib/srv/uacc/uacc_linux.go:186:50: utmpPath redeclared in this block
        lib/srv/uacc/uacc_linux.go:186:22: other declaration of utmpPath
lib/srv/uacc/uacc_linux.go:186:60: wtmpPath redeclared in this block
        lib/srv/uacc/uacc_linux.go:186:32: other declaration of wtmpPath
```
I can rename it, but at this point, this sounds a bit "forced".

Reply:
Ah yeah, good call. Not a big deal.